controllers/constants/constants.go

```diff
@@ -32,6 +32,9 @@ const (
 	NewConnectionType = "newConnection"
 	HeartbeatType     = "heartbeat"
 	EndConnectionType = "endConnection"
+
+	MonitoringNamespace      = "openshift-monitoring"
+	MonitoringServiceAccount = "prometheus-k8s"
```

Comment:
what are these hard coded values for?

Reply:
Constants for monitoring namespace and monitoring role, they were already there, I just moved them to the constant file since they are now used at different places.